charts/gateway-helm/crds/generated/gateway.envoyproxy.io_backendtrafficpolicies.yaml

```diff
@@ -412,15 +412,29 @@ spec:
                       ConsistentHash defines the configuration when the load balancer type is
                       set to ConsistentHash
                     properties:
+                      header:
+                        description: Header configures the header hash policy when
+                          the consistent hash type is set to Header.
+                        properties:
+                          header_name:
```

Comment:
I can't get this to regenerate –`header_name` should be `name` after updating `loadbalancer_types.go`. I looked through `make help` and ran a variety of `make helm-package` and `make generate-artifacts` but no dice. What's happening here?

Reply:
Never mind, figured it out. `make manifests`